src/assignement2/Starter.java

```diff
@@ -0,0 +1,119 @@
+package assignement2;
+
+import java.util.Scanner;
+import java.util.Set;
+/*
+ * This class takes choice from the user
+ * and try to class various method accordingly
+ * It has us as a set of user
+ */
+
+public class Starter {
+
+    //    variable used to store the user details
+    private Set<User> us;
+
+    /*This is a default constructor
+     * it instanties set from file
+     */
+    public Starter() {
+        this.us = new StorgeClass().getUsers();
+    }
+
+    /*
+     * This method print choice and there details
+     * @return choice of user
+     */
+
+    private int showOptions() {
+        System.out.println("Here is options number and task they perform on pressing them");
+        System.out.println("1. Add user details");
+        System.out.println("2. Display user details");
+        System.out.println("3. Delete user details");
+        System.out.println("4. Save user details");
+        System.out.println("5. Exit");
+        try {
+            final Scanner sc = new Scanner(System.in);
+            return sc.nextInt();
+        } catch (NumberFormatException e) {
+            return Constants.ch_5;
+        }
```

Comment:
what is this ?

Reply:
This will return 5 choice by default when wrong input of choice is made